<ui version="4.0" >
 <class>BAntViewWidget</class>
 <widget class="QWidget" name="BAntViewWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BAnts</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGLWidget" native="1" name="myGLWidget" />
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>TODO: make me usefull</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="myAntAddButton" >
        <property name="text" >
         <string>Add Ant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="myPaintGroupBox" >
        <property name="title" >
         <string>Painter</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QCheckBox" name="mySmellCheck" >
           <property name="text" >
            <string>Smell</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
           <property name="tristate" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="myAntCheck" >
           <property name="text" >
            <string>Ants</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
           <property name="tristate" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="myGoalCheck" >
           <property name="text" >
            <string>Goals</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
           <property name="tristate" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="myAntialiasingCheck" >
           <property name="text" >
            <string>Antialiasing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="myAntLabel" >
          <property name="text" >
           <string>Ants:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="myAntCountLabel" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>178</width>
          <height>141</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGLWidget</class>
   <extends>QWidget</extends>
   <header>qglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
